client/my-sites/github-deployments/deployment-run-logs/deployments-run-item.tsx

```diff
@@ -65,7 +71,19 @@ export const DeploymentsRunItem = ( { run }: DeploymentsListItemProps ) => {
 					<DeploymentDuration run={ run } />
 				</td>
 				<td>
-					<Button plain onClick={ handleToggleExpanded }>
+					<Button
+						plain
+						onClick={ () => {
+							if ( ! expanded ) {
+								dispatch(
+									recordTracksEvent( 'calypso_hosting_github_log_entry_expanded', {
+										row_number: rowNumber,
```

Comment:
What's the reason behind this `row_number` property?

Reply:
Mostly to check if they are interested in old log records. This relates to discussions about how many log records to keep.